### PR TITLE
Ready for inspection: spglib

### PR DESCRIPTION
**Pull request by jochym**
Spglib is a library for finding and handling crystal symmetries.

### CODE REVIEW

File: recipes/spglib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+build:
+  skip: true  # [win and (py27 or py34)]
+  number: 0
+  script: 
+    - python setup.py install --single-version-externally-managed --record record.txt 
+
+requirements:
+  build:
+    - python
+    - numpy
```
Comment (scopatz) [inline recipes/spglib/meta.yaml]: If numpy is a build dependency, you need to pin to `numpy x.x`

Comment (jochym) [inline recipes/spglib/meta.yaml]: @scopatz why is this necessary? Something gets linked in? Even for poorly python package? 
I naturally will do that - I just want to understand the reasons?

Comment (scopatz) [inline recipes/spglib/meta.yaml]: Unfortunately, numpy's ABI is a special feature of conda (like Python versions). I don't know why the `x.x` is used instead of `1.11` or some other concrete number, but I imagine that it has to do with the fact that conda recipes are templates that are filled in.  Others probably know more.

Comment (jochym) [inline recipes/spglib/meta.yaml]: Done

File: recipes/spglib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - python setup.py install --single-version-externally-managed --record record.txt 
+
+requirements:
+  build:
+    - python
+    - numpy x.x
+    - setuptools
+  run:
+    - python
+    - numpy
```
Comment (scopatz) [inline recipes/spglib/meta.yaml]: I think you need the x.x here too

**Comment by jochym** [comment]
I think this is ready. How about merging @scopatz ? He asked politely ;)

**Comment by scopatz** [comment]
Thanks!
